### ISSUE TITLE
Check out github Projects

### ISSUE THREAD

**Issue by jhornert** [genevaers/Community#129]

**Comment by jhornert** [comment]
github document on Project

https://github.com/features/project-management/

**Comment by hammyau** [comment]
Justification/ideas for using Github Projects rather than the Jira.

- Publicly visible
- One system - we do not have many developers and this will mean a reduced learning curve.
- Inherently links to the Github issues, checkins and PRs. 
- Responsive - I find the Jira to be pretty slow.

We need to consider workflows and how to structure the Projects.
## Where do issues come from/types. (-> Labels)

- Todos/task - not a direct development thing
- Questions
- Discussions (just list this one :-) )
- Bug
- Enhancement

Within a Project the canned templates have some easy setups. The Kanban with review looks okay/
I think the simpler the better.
We can also add labels to highlight state or development area ( a project can span repositories )
The boards automatically filter on labels or combinations of them.

There are also milestones... another time related grouping mechanism for issues.

## Project permissions

Need to figure out who can move the issues around. Do we want to limit?
Certainly to some extent. Any changes are tracked anyway.

**Comment by hammyau** [comment]
Another thing ... the projects etc can have an associated wiki.
That may be a place to keep the internal documentation rather than via the Gatsby beast?

**Comment by emorrow38** [comment]
I got the impression at the Scrum I attended that JIRA was used as an internal tools for the GERS project, whereas GitHub would be used as the universal public tool.  I see that as sensible - the public issues will be different to our internal issues, because we have to juggle with the paid product (SAFR) as well as GERS.  Surely we are concerned with the transition from SAFR to GERS where as the public only wants to know about GERS and what they can get.
So let's go with both tools - JIRA for internal and GitHub for public issues.
